edu_edfi_airflow/dags/earthbeam_dag.py

```diff
@@ -31,6 +31,26 @@ class EarthbeamDAG:
             type="boolean",
             description="If true, passes `--force` flag to Earthmover and Lightbeam"
         ),
+        "send": Param(
+            default=True,
+            type="boolean",
+            description="If true, passes `--send` flag to Earthmover and Lightbeam"
```

Comment:
earthmover doesn't have a send/validate arg, right? so these are only applicable to lightbeam? Can we name the params accordingly?